cpp/tests/io/parquet_chunked_reader_test.cu

```diff
@@ -377,105 +377,105 @@ TEST_F(ParquetChunkedReaderTest, TestChunkedReadWithString)
   }
   {
     auto const [result, num_chunks] = chunked_read(filepath_with_nulls, 0);
-    EXPECT_EQ(num_chunks, 1);
+    // EXPECT_EQ(num_chunks, 1);
```

Comment:
Sorry I missed these last time, these three checks (L380, 385, 390) are also being read with `zero` limit and so the number of chunks produced should be 1 and the test can be reenabled